src/main/java/com/google/devtools/build/lib/skyframe/config/ParsedFlagsFunction.java

```diff
@@ -65,7 +65,8 @@ public SkyValue compute(SkyKey skyKey, Environment env)
     }
     // The StarlarkOptionsParser needs a native options parser just to inject its Starlark flag
     // values. It doesn't actually parse anything with the native parser.
```

Comment:
update this comment to mention the conversion context

Reply:
Done